database/models/User.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable space-before-function-paren */
```

Comment:
Please change the name of this file, it should start with a lowercase since it doesn't export a class. Thank you